docs/reference/announcements-release-notes/880/880-announcements.md

```diff
@@ -303,16 +303,23 @@ With the Camunda 8.8 release, Camunda Java Client and Camunda Spring Boot SDK re
 
 The `CamundaClient` replaces the `ZeebeClient`, offering the same functionality and adding new capabilities.
 
-The Camunda Spring Boot SDK is based on Spring Boot 3.5, see [version compatibility matrix](/apis-tools/spring-zeebe-sdk/getting-started.md#version-compatibility).
-
 :::note
 
+- The new `CamundaClient` uses REST as the default communication protocol, explicitly use the configuration option `preferRestOverGrpc=false` to switch to gRPC as the default protocol. (Note: job streaming is only supported via gRPC, but can be used alongside REST for other operations)
 - If you need to continue using the old `ZeebeClient`, you can use the new version 8.8 `CamundaClient` artifact without issues, as it still contains the related `ZeebeClient` classes. Those classes are marked as deprecated, so you can easily spot code you need to adjust to the `CamundaClient`.
 - The old `zeebe-client-java` artifact is now relocation-only, so your build system is redirected to the new `camunda-client-java` artifact. We will discontinue the old artifact in version 8.10 and recommend using the new one.
 - The Zeebe Java client will not be developed further and will only receive bug fixes while version 8.7 is officially supported.
 
 :::
 
+The Camunda Spring Boot SDK is based on Spring Boot 3.5, see [version compatibility matrix](/apis-tools/spring-zeebe-sdk/getting-started.md#version-compatibility).
+
+:::warning
+
+Breaking Change: The new Spring SDK relies on the `CamundaClient` and therefore uses REST as the default communication protocol. If you want to continue using gRPC by default, you need to explicitly set `preferRestOverGrpc=false` in your configuration.
```

Comment:
@megglos, in the context of the split of artifacts that you're working on, we don't need that, right?

Reply:
The Spring Boot Starter should currently support both clients, so if you use ZeebeClient in your code, you'll get it. Is this statement based on some test where it didn't work?

With the split https://github.com/camunda/camunda/issues/37628 we ideally end up with a starter for each client.

Reply:
@jonathanlukas mentioned this further above. Is your suggestion here that this notice can be removed @megglos ?

Reply:
So as of now, nothing would break if you switch, if you used the ZeebeClient you still have it.

If you switch to the CamundaClient with using the Spring Boot Starter, you'll get the new default client behavior. So I would frame it as an info referring to the fact that the default behavior depends on the client used.

```suggestion
:::info

The new Camunda Spring Boot Starter provides the `CamundaClient` when requested. The `CamundaClient` uses REST as the default communication protocol, while the deprecated `ZeebeClient` still prefers gRPC. If you want to continue using gRPC by default with the `CamundaClient`, you need to explicitly set `camunda.client.prefer-rest-over-grpc: false` in your Spring configuration.
```